libraries/common/src/test/java/com/uber/okbuck/example/common/ExampleUnitTest.java

```diff
@@ -25,5 +24,8 @@ public void addition_isCorrect() throws Exception {
     public void resource_loading() {
         Context context = (Context) RuntimeEnvironment.application;
         assertEquals(context.getResources().getString(R.string.app_name), "Common");
+        assertEquals(context.getResources().getColor(
+            com.uber.okbuck.example.parcelable.R.color.fooColor), 0xcccccc);
```

Comment:
This is probably not the right example. This specific problem goes away when you use the Right R class here.

Can you modify this change to repro your case i.e have another module refer to the resource in its main code and this module invokes that piece of code in it's test?